src/test/java/com/easyrest/tests/Listeners.java

```diff
@@ -8,12 +8,17 @@
 
 public class Listeners extends BaseTest implements ITestListener {
 
+    public void onTestStart(ITestResult result) {
+        test = extent.createTest(result.getTestClass().getName() + " : " +
+                result.getMethod().getMethodName());
+    }
+
    public void onTestSuccess(ITestResult result) {
-// TODO Auto-generated method stub
-        System.out.println("Success of test cases and its details are : "+result.getName());
+       System.out.println("Test passed: " + result.getMethod().getMethodName());
```

Comment:
Why did you use System.out.println when we have Extent Reports.

Reply:
> Why did you use System.out.println when we have Extent Reports.

Because it is okay. Why we don t need to use? What we can use instead?

Reply:
@martadiachok I think you can use the `test `object to log info to the extent report.

Reply:
> @martadiachok I think you can use the `test `object to log info to the extent report.

fix it